imls-directus/docker-compose.yml

```diff
@@ -1,74 +1,25 @@
 version: "3"
 services:
```

Comment:
I would also prefer a new directory for this docker compose file: `imls-backend` perhaps?